docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
     gateway:
         ports:
             - 8080:8080
-        image: openfaas/gateway:0.8.5
+        image: openfaas/gateway:0.8.6
```

Comment:
Can you sync to latest? Not sure we want these changes in this PR

Reply:
@ericstoekl :+1: sure no worries, i'll leave it at 0.8.5

Reply:
We have released `0.8.6` already. This would be in 0.8.7 or 0.9.0 but FYI the release tag comes after the code is merged and promoted. That generally means: merge, release - update compose files.